packages/std/src/testing/mock.rs

```diff
@@ -358,6 +358,7 @@ pub fn mock_env() -> Env {
 
 /// Just set sender and funds for the message.
 /// This is intended for use in test code only.
+#[deprecated(note = "This is inconvenient and unsafe. Use message_info instead.")]
```

Comment:
There's one more usage of `mock_info` in the `coins` doctest that should be replaced.

Reply:
Thanks, fixed